my-cypress-project/cypress/pages/realWordAppPage.js

```diff
@@ -0,0 +1,117 @@
+export default class LoginPage {
+    constructor() {
+        this.userInfo = {
+            firstName: "Olena",
+            lastName: "Gofman",
+            username: "OlenaGofman",
+            password: "secret",
+            confirmPassword: "secret"
+        }
+
+        this.date = new Date().getTime();
+    }
+    openRealWorldApp() {
+        cy.visit('http://localhost:3000/')
```

Comment:
This can be in the base_url
https://filiphric.com/cypress-basics-using-baseurl
https://docs.cypress.io/guides/references/best-practices#Setting-a-Global-baseUrl